src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml

```diff
@@ -93,15 +93,10 @@
                             <div class="tab-pane" id="python-help">
                                 <ol>
                                     <li>
-                                        Install apistubgen.
-                                        <code>pip install git+"https://github.com/Azure/azure-sdk-tools.git#subdirectory=packages/python-packages/api-stub-generator&egg=api-stub-generator"</code>
+                                        Generate wheel for the package
```

Comment:
Can you expand a little bit, maybe a simple command line sample?

Reply:
Done.